tests/cephadm/ansible_wrapper/cephadm_bootstrap/bootstrap-with-custom-ssh.yaml

```diff
@@ -8,7 +8,6 @@
       cephadm_bootstrap:
         mon_ip: "{{ mon_ip }}"
         image: "{{ image | default(False) }}"
-        dashboard: "{{ dashboard | default(True) }}"
```

Comment:
Why this removed ? Is this parameter unsuppported ?

If test case require to disable this then set default value to `False`

Reply:
This parameter is causing failure in the tests since it requires "dashboard-user". Bug has been raised for the same (see description)
It is not necessary to provide this parameter since it enabled by default. To disable it we can use "skip-dashboard" parameter instead.
Hence for the CI we can remove this parameter completely.